### ISSUE TITLE
Check for duplicate candidates/parties/positions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halalaninitiative/halalan#16]
```
Check for already existing candidates/parties/positions when manually
adding via web interface.
```

Original issue reported on code.google.com by `djclue...@gmail.com` on 15 Feb 2008 at 5:37

**Comment by GoogleCodeExporter** [comment]
```
Changed status to Started.
```

Original comment by `djclue...@gmail.com` on 15 Feb 2008 at 6:18
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in r288.
```

Original comment by `djclue...@gmail.com` on 15 Feb 2008 at 6:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `toni...@gmail.com` on 16 Feb 2008 at 12:49
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
There is still no checking when editing data.
```

Original comment by `waldemar...@gmail.com` on 16 Feb 2008 at 7:02
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r311.
```

Original comment by `djclue...@gmail.com` on 17 Feb 2008 at 2:50
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Removed checking if candidate/party/position exists when that
candidate/party/position is being edited.
```

Original comment by `waldemar...@gmail.com` on 17 Feb 2008 at 4:11

**Comment by GoogleCodeExporter** [comment]
```
Fix verified.
```

Original comment by `djclue...@gmail.com` on 17 Feb 2008 at 4:26
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f83a80f791.
```

Original comment by `daru...@gmail.com` on 4 Sep 2010 at 1:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `waldemar...@gmail.com` on 7 Feb 2011 at 9:16
- Changed state: **Verified**
